inc/inventoryrulelocation.class.php

```diff
@@ -153,11 +153,7 @@ function executeActions($output, $params) {
                         $entities_id = $_SESSION["plugin_fusioninventory_entity"];
                      }
                      $res = Dropdown::importExternal(
-                             getItemTypeForTable(
-                                     getTableNameForForeignKeyField(
-                                             $action->fields['field'])),
-                             $res,
-                             $entities_id);
+                        getItemTypeForTable(getTableNameForForeignKeyField($action->fields['field'])), $res, $entities_id);
```

Comment:
why not to use getItemtypeForForeignKeyField ?

Reply:
Same as former answer...